public/app/features/datasources/settings/ButtonRow.tsx

```diff
@@ -17,6 +17,9 @@ const ButtonRow: FC<Props> = ({ isReadOnly, onDelete, onSubmit, onTest }) => {
       <LinkButton variant="secondary" fill="outline" href={`${config.appSubUrl}/datasources`}>
         Back
       </LinkButton>
+      <LinkButton variant="secondary" fill="solid" href={`${config.appSubUrl}/explore`}>
+        Explore your data
```

Comment:
Could you name this just Explore? Thanks

Reply:
sure